FirebaseAuthUI/README.md

```diff
@@ -319,3 +319,51 @@ parent classes. For example:
 
 Refer to the Objective-C and Swift samples for examples of how you can customize
 these views.
+
+## Handling auto-upgrade of anonymous users
+
+Enabling auto-upgrade of anonymous users increases the complexity of your auth
+flow by adding several more edge cases that need to be handled. As opposed to
+normal auth, which only involves one step, auto-upgrade presents three steps
+with four possibilities total:
+- At app launch, anonymously authenticate the user. User state can be
+  accumulated on the anonymous user and linked to the non-anonymous account
+  later.
+- At some point in your app, present the auth flow and authenticate the user
+  using a non-anonymous auth method.
+- If the user signs in to a new account, the anonymous account and the new
```

Comment:
It would be nice (though not strictly required) for these bullets all to be parallel/identical in structure.

The first two go *At clause, command verb*; can the last two bullets be structured that way too?  Or at least a "When x happens, command verb" style?

Reply:
Done.